src/vmm/src/device_manager/persist.rs

```diff
@@ -239,7 +239,7 @@ impl<'a> Persist<'a> for MMIODeviceManager {
                                   event_manager: &mut EventManager|
          -> Result<(), Self::Error> {
             dev_manager
-                .slot_sanity_check(slot)
+                .mmio_device_info_sanity_check(slot)
```

Comment:
nit: for consistency with the rest of your changes:
```suggestion
                .mmio_device_info_sanity_check(device_info)
```